pkg/tagger/README.md

```diff
@@ -17,6 +17,12 @@ The tagger is also available to python checks via the `tagger` module exporting
 the `get_tags()` function. This function accepts the same arguments as the Go `Tag()`
 function, and returns an empty list on errors.
 
+**Note:** for performance reasons, the slices returned by the tagger are cached and
```

Comment:
I don't think the performance required it warrants the risk. The tagger should expose a safe API. If we need it to be fast in specific cases I'm okay with @mfpierre 's proposal of having an explicit unsafe method, which we'll introduce if we need to, but we probably don't right now.